Library/Homebrew/cmd/tests.rb

```diff
@@ -1,9 +1,34 @@
 require "fileutils"
+require "official_taps"
 
 module Homebrew
   def tests
+    ENV["HOMEBREW_NO_ANALYTICS_THIS_RUN"] = "1"
+    Homebrew.install_gem_setup_path! "bundler"
+
+    if ARGV.include? "--official-cmd-taps"
+      OFFICIAL_CMD_TAPS.each do |org, repo|
+        tap = "#{org}/#{repo}"
+        tap_dir = HOMEBREW_LIBRARY/"Taps/#{org}/homebrew-#{repo}"
+
+        # It has no tests :(
+        next if "#{tap}" == "homebrew/services"
```

Comment:
Should we maybe instead check for the presence of a `Rakefile` instead of hard-coding this? Though that would uselessly install the tap and then do nothing with it, if there is none.